mmv1/third_party/terraform/utils/bootstrap_utils_test.go

```diff
@@ -358,6 +359,136 @@ func BootstrapServicePerimeterProjects(t *testing.T, desiredProjects int) []*clo
 	return projects
 }
 
+func removeContainerServiceAgentRoleFromContainerEngineRobot(t *testing.T, project *cloudresourcemanager.Project) {
+	config := BootstrapConfig(t)
```

Comment:
Since you are using `BootstrapConfig`, I don't believe the requests being made in this function will get recorded or replayed by the VCR. That means the requests will be made for every replay (ie. multiple per PR). You should be able to test with multiple GCB runs in this PR.

If the requests aren't recorded, it might be worth checking the policy first and only calling "Set" if it needs to be updated. That would also make it more like our other bootstrap functions, so you could potentially use that pattern more explicitly, ie. `BootstrapProjectWithInvalidContainerEngineRobot()`. This assumes that you never need to reset this project to the original state.

Reply:
I have made this function only set the policy when changes are required.